tests/integration/security/webhook/config/sidecar-injector-webhook.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     release: istio
 webhooks:
   - name: sidecar-injector.istio.io
+    admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
I am a bit confused how this works. Does this mean the api server will send admissionReview/v1 first, then fallback to v1beta1 if it fails or something? Why do we want both here?

Reply:
>Does this mean the api server will send admissionReview/v1 first, then fallback to v1beta1 if it fails or something?

Yes, can be found in api docs

I think it is to be compatible with older webook which only serves v1beta1 admissionReview, currently we serve v1beta1 in istiod

Reply:
Yup